packages/components/src/context-actions/ContextActionUtils.ts

```diff
@@ -17,6 +17,7 @@ export interface ContextAction {
   icon?: IconDefinition | React.ReactElement;
   iconColor?: string;
   shortcut?: Shortcut;
+  shortcutText?: string;
```

Comment:
While we're in here, should add some JSDocs for the new property (ideally we'd have JSDocs for every prop, but adding just for the new property is good for now). Just explaining this will display text for a shortcut in the menu, but does not actually wire up the shortcut.